.github/workflows/create-branch-on-issue.yml

```
@@ -0,0 +1,52 @@
+          BRANCH="${LABEL}/${ISSUE_NUM}-${SLUG}"
+          echo "Creating branch: $BRANCH"
+
+          git checkout -b "$BRANCH"
+          git push origin "$BRANCH"
+
+          # Issue에 브랜치명 코멘트 자동 등록
+          gh issue comment $ISSUE_NUM \
+            --body "🌿 브랜치가 생성되었습니다: \`$BRANCH\`
+            
+작업을 시작하려면:
+\`\`\`bash
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)